Fix MergeResponse Call for Keyword Generations

positiveVibes4all opened the pull request:
## Summary 
MergeResponse function was updated, and this PR corrects access to a function that was needed for keyword generations.

decrapifier commented:
<!--
  1 failure:  Danger failed to ...
  0 warning: 
  
  1 markdown notices
  DangerID: danger-id-Danger;
-->

<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Danger failed to run `dangerfile.js`.
  </td>
    </tr>
  </tbody>
</table>



## Error SyntaxError
```
Cannot use import statement outside a module
dangerfile.js:3
import fs from 'fs';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at Object.compileFunction (node:vm:360:18)
    at wrapSafe (node:internal/modules/cjs/loader:1055:15)
    at Module._compile (node:internal/modules/cjs/loader:1090:27)
    at requireFromString (/home/runner/work/contexture/contexture/node_modules/danger/node_modules/require-from-string/index.js:28:4)
    at /home/runner/work/contexture/contexture/node_modules/danger/distribution/runner/runners/inline.js:157:68
    at step (/home/runner/work/contexture/contexture/node_modules/danger/distribution/runner/runners/inline.js:52:23)
    at Object.next (/home/runner/work/contexture/contexture/node_modules/danger/distribution/runner/runners/inline.js:33:53)
    at /home/runner/work/contexture/contexture/node_modules/danger/distribution/runner/runners/inline.js:27:71
    at new Promise (<anonymous>)
    at __awaiter (/home/runner/work/contexture/contexture/node_modules/danger/distribution/runner/runners/inline.js:23:12)
```
### Dangerfile
```
---------------------^
```
  
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against e440573dfedf60871b8397914a44c4afc9b2b47a
</p>

daedalus28 reviewed (approved):
Good stuff

Commit:
Update Code Per Recent Changes

- A recent update altered the way we access extend for mergeResponse function. This corrects this in the case of keyword generations.

Commit:
Version Bump


Remove Log

lint-action committed:
Fix code style issues with Prettier
